packages/now-node/src/index.ts

```diff
@@ -193,7 +193,7 @@ async function compile(
   });
 
   debug('traced files:');
-  debug('\t' + fileList.join('\n\t'));
+  console.log('\t' + fileList.join('\n[now-builder-debug] \t'));
```

Comment:
Let's delete these two lines. Its never relevant to the user. These were more for debugging ourselves.

Reply:
Let's also remove the `debug('compiling es module file ' + path);` line too

Reply:
And also remove `debug('compiling typescript file ' + relPath);`

Reply:
Removed 🙌